gap/io.gi

```diff
@@ -821,8 +826,14 @@ function(s)
                   "the input string <s> should be non-empty,");
   fi;
 
-  # Check for the special '+' character
-  if s[1] <> '+' then
+  # Check for the special '&' character (or the deprecated '+')
+  if s[1] = '&' then
+    legacy := false;
+  elif s[1] = '+' then
+    legacy := true;
+    Info(InfoDigraphs, 1, "Digraph6 strings beginning with '+' are deprecated");
+    Info(InfoDigraphs, 1, "and may not be recognised by other programs.");
```

Comment:
I think this warning needs to be more forceful/explicit. You could read this and think that the only thing that changed was that the '+' has been changed to a '&'. But the actual encoding is fundamentally different.

Maybe something like

> Digraph6 strings that begin with '+' use an old specification of the Digraph6 format that is incompatible with the current, and now standard, specification. These old strings can still be read by the Digraphs package, but it is unlikely that they can be read correctly by other programs. We recommend that you re-encode your digraphs with the new format.